vnext/include/ReactUWP/XamlView.h

```diff
@@ -30,4 +31,22 @@ inline void SetTag(XamlView view, winrt::IInspectable tag)
   SetTag(view, tag.as<winrt::IPropertyValue>().GetInt64());
 }
 
+inline bool IsValidTag(winrt::IPropertyValue value)
+{
+  assert(value);
+  return (value.Type() == winrt::PropertyType::Int64);
+}
+
+inline int64_t GetTag(winrt::IPropertyValue value)
+{
+  assert(value);
+  return value.GetInt64();
+}
+
+inline winrt::IPropertyValue GetPropertyValue(winrt::FrameworkElement fe)
```

Comment:
>GetPropertyValue [](start = 29, length = 16)

This function name doesn't tell the reader what property value is being returned, I suggest renaming to GetTagAsPropertyValue().

Reply:
Or

---
In reply to: [295445849](https://github.com/microsoft/react-native-windows/pull/2623#discussion_r295445849) [](ancestors = 295445849)

Reply:
Which would then mean IsValidTag() needs to be something like IsTagValidViewId().

---
In reply to: [295518929](https://github.com/microsoft/react-native-windows/pull/2623#discussion_r295518929) [](ancestors = 295518929,295445849)

Reply:
I keep Tag in this PR. Tag is used everywhere in this project, and prefer have another task to rename them if needed.